msteams-platform/concepts/bots/bots-create.md

```diff
@@ -16,7 +16,7 @@ See the [Bot Framework Documentation](https://docs.botframework.com/en-us/) for
 
```

Comment:
Excellent suggestion, @arelend. Thanks for the issue and PR! Also, thanks for submitting the PR against master branch instead of the default live. That saves a few manual steps for us.